quickstart/BUILD

```diff
@@ -17,9 +17,9 @@ package(default_visibility = ["//visibility:public"])
 licenses(["notice"])  # Apache 2.0
```

Comment:
In order to keep this BUILD file as simple as possible for our first-time users, I think we could remove the `package` statement.

Reply:
Fixed.